src/ocaml/preprocess/408/parser_raw.mly

```diff
@@ -487,15 +487,15 @@ let class_of_let_bindings ~loc lbs body =
       lbs.lbs_bindings
   in
     (* Our use of let_bindings(no_ext) guarantees the following: *)
-    assert (lbs.lbs_extension = None);
+    (* assert (lbs.lbs_extension = None); *)
```

Comment:
If that assertion is incorrect, is it due to something specific in merlin, or is it something we should report upstream?

Reply:
It is unlikely that the parser recovery introduces an extension here, but in theory that might happen. It is ok to leave the assertion if you feel better about that.